activemq-karaf/src/main/resources/features-core.xml

```diff
@@ -31,6 +31,7 @@
         <bundle dependency="true">mvn:org.jvnet.jaxb2_commons/jaxb2-basics-runtime/${jaxb-basics-version}</bundle>
         <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.jaxb-impl/${jaxb-bundle-version}</bundle>
         <bundle>mvn:org.apache.commons/commons-pool2/${commons-pool2-version}</bundle>
+        <bundle dependency="true">mvn:org.apache.zookeeper/zookeeper/${zookeeper-version}</bundle>
```

Comment:
Although the original commit did remove it, putting this back for the actual release seems wrong. It seems surprising it would actually be related to the API change, so assuming it wasnt really then either omitting it here or adding another commit to take it back out would seem appropriate.

Reply:
Ah, already discussed while I was delayed from completing my review hehe.